view/theme/frio/css/style.css

```diff
@@ -76,6 +76,12 @@ blockquote {
 .fakelink {
 	cursor: pointer;
 }
+.createforum {
+	padding: 25px;
+	cursor: pointer;
+	padding-top: 5px;
+}
```

Comment:
If you want to give a button appearance to the forum creation button, you can use the `btn btn-primary` classes instead of custom CSS.

Reply:
Would you prefer a button over a link?

Reply:
By adding padding and changing the cursor look, you’ve effectively changed the link into a button, I’m just picking up what you put down.

If it’s meant to be a link, the CSS isn’t necessary.

Reply:
https://friendica.myportal.social/photos/anubis2814/image/6279611596646b787465252720731759
Here is what it looks like on screen, but with a little more padding above.  Let me know if you like how it looks or it should be an actual button, and if I should use the FA plus sign on it.

Reply:
Thank you for the screenshot, now that I'm looking at it I feel like the button to create a new forum should be a plus icon in the same place where the "Edit" icon is in the "Groups" widget, at the top right.

Reply:
Oh yeah that will look much more sleek and minimalistic.  I'll do that.